src/helow/reporter/views.py

```diff
@@ -13,3 +16,27 @@ class CreateIncidentReportView(generics.ListCreateAPIView):
 class DetailIncidentReportView(generics.RetrieveUpdateAPIView):
     queryset = IncidentReport.objects.all()
     serializer_class = serializers.CreateIncidentReportSerializer
+
+
+class IncidentListView(generics.ListAPIView):
+    serializer_class = serializers.CreateIncidentReportSerializer
+    pagination_class = PageNumberPagination
+
+    def get_queryset(self):
+        queryset = IncidentReport.objects.all()
+        location = self.request.query_params.get('location')
+        reported_date = self.request.query_params.get('reported_at')
+        incident_type = self.request.query_params.get('incident_type')
+        status = self.request.query_params.get('is_status_open')
+        if status == 'true':
+            queryset = queryset.filter(is_status_open=True)
+        elif status == 'false':
+            queryset = queryset.filter(is_status_open=False)
+        elif location:
+            queryset = queryset.filter(location__name=location)
+        elif reported_date:
+            queryset = queryset.filter(reported_at=reported_date)
+        elif incident_type:
+            queryset = queryset.filter(incident_type__label=incident_type)
+        return queryset.order_by('-reported_at')
+    
```

Comment:
Please remove trailing whitespace.

Reply:
@haffs0 nice job on your PR. Please take out some time to effect the reviews i marked to enable your PR to be acceptable to cadacy.

Most importantly, before you do these, checkout to develop on your local and do 'git pull', then checkout to your feature branch and do 'git merge develop'. There are many updates your local copy is missing. Merging this would result to a conflict.

Kindly effect this changes and push again. Thank you.